report/src/main/webapp/assets/js/newComparison/newComparison.js

```diff
@@ -0,0 +1,698 @@
+window.initNewComparison = function (window, document, undefined) { return function(event) {
```

Comment:
Indents seem to be all over the place in this file. Please fix them for readability.